### PR TITLE
Move text-direction discussion from Wiki to new /docs area

### PR DESCRIPTION
**Pull request by gkellogg**
... with reference from the README.

Once merged, the Wiki can be turned into a reference to /docs/text-direction.md.

### CODE REVIEW

File: docs/text-direction.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+According to the [W3C Process Document](https://www.w3.org/2021/Process-20211102/):
+
+> All reports, publications, or other deliverables produced by the group for public consumption should follow best practices for **internationalization** and for accessibility to people with disabilities. Network access to W3C-controlled domains may be assumed.
+
+One thing that wasn't given as much attention in earlier Working Groups (including RDF 1.1) was support for describing the base paragraph direction that text should should use for presentation. Although Unicode provides some information about the directionality of individual code points and a way to estimate base paragraph direction when no other information is available (see [First-string property detection](https://w3c.github.io/string-meta/#firststrong), there are still many cases in which this results in incorrect results. The requirements here are captured  in https://github.com/w3c/rdf-concepts/issues/9 on RDF Concepts, where much of this information has been gathered. Of course, changes potentially will impact other specifications.
```
Comment (aphillips) [inline docs/text-direction.md:5]:
```suggestion
One thing that wasn't given as much attention in earlier Working Groups (including RDF 1.1) was support for describing the base paragraph direction that text should should use for presentation. Although Unicode provides some information about the directionality of individual code points and a way to estimate base paragraph direction when no other information is available (see [First-strong property detection](https://w3c.github.io/string-meta/#firststrong), there are still many cases in which this results in incorrect results. The requirements here are captured  in https://github.com/w3c/rdf-concepts/issues/9 on RDF Concepts, where much of this information has been gathered. Of course, changes potentially will impact other specifications.
```

**Review by aphillips — COMMENTED** [review]
LGTM (we can do more later)

File: README.md [hunk 2]
```diff
@@ -7,6 +7,14 @@ For general issues about RDF-star and the RDF/SPARQL 1.2 family of specification
 you can [raise an issue](https://github.com/w3c/rdf-star-wg/issues) on this repository,
 or send an email to the [public mailing list](https://lists.w3.org/Archives/Public/public-rdf-star-wg/).
 
-For more targetted discussion on one of the deliverables, use [the corresponding repository](https://www.w3.org/groups/wg/rdf-star/tools).
+For more targeted discussion on one of the deliverables, use [the corresponding repository](https://www.w3.org/groups/wg/rdf-star/tools).
 
+## Working Group documents
 
+In addition to the related repositories referenced as deliverables above. Contributors should reference the [Editor's Guide](https://github.com/w3c/rdf-star-wg/wiki/Editor's-guide) for general practices in editing specifications, submitting Pull Requests and generally working with ReSpec.
```
Comment (TallTed) [inline README.md:14]:
```suggestion
In addition to the related repositories referenced as deliverables above. Contributors should reference the [Editor's Guide](https://github.com/w3c/rdf-star-wg/wiki/Editor's-guide) for general practices in editing specifications, submitting Pull Requests, and generally working with [ReSpec](https://respec.org/docs/).
```

File: README.md [hunk 3]
```diff
@@ -7,6 +7,14 @@ For general issues about RDF-star and the RDF/SPARQL 1.2 family of specification
...
+
+Note that per the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/) the group is only able to accept non-trivial contributions from Working Group members. However, comments and minor update suggestions from others are welcome.
```
Comment (TallTed) [inline README.md:16]:
```suggestion
Note that per the [W3C Patent Policy](https://www.w3.org/Consortium/Patent-Policy-20200915/), the group is only able to accept non-trivial contributions from Working Group members. However, comments and minor update suggestions from others are welcome.
```

File: docs/text-direction.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
+According to the [W3C Process Document](https://www.w3.org/2021/Process-20211102/):
+
+> All reports, publications, or other deliverables produced by the group for public consumption should follow best practices for **internationalization** and for accessibility to people with disabilities. Network access to W3C-controlled domains may be assumed.
+
+One thing that wasn't given as much attention in earlier Working Groups (including RDF 1.1) was support for describing the base paragraph direction that text should should use for presentation. Although Unicode provides some information about the directionality of individual code points and a way to estimate base paragraph direction when no other information is available (see [First-strong property detection](https://w3c.github.io/string-meta/#firststrong), there are still many cases in which this results in incorrect results. The requirements here are captured  in https://github.com/w3c/rdf-concepts/issues/9 on RDF Concepts, where much of this information has been gathered. Of course, changes potentially will impact other specifications.
```
Comment (TallTed) [inline docs/text-direction.md:5]:
```suggestion
One thing that wasn't given as much attention in earlier Working Groups (including RDF 1.1) was support for describing the base paragraph direction that text should should use for presentation. Although Unicode provides some information about the directionality of individual code points and a way to estimate base paragraph direction when no other information is available (see [First-strong property detection](https://w3c.github.io/string-meta/#firststrong), there are still many cases in which this delivers incorrect results. The requirements here are captured in [issue #9 on RDF Concepts](https://github.com/w3c/rdf-concepts/issues/9), where much of this information has been gathered. Of course, changes will potentially impact other specifications.
```

File: docs/text-direction.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+
+The [Internationalization Best Practices for Spec Developers Note](https://www.w3.org/TR/international-specs/) has a whole section devoted to [Text direction](https://www.w3.org/TR/international-specs/#text_direction), and this is a checklist item for the internationalization review all specs need to go through. When this was performed for JSON-LD 1.1, it came up that the underlying data model (RDF) had no way to indicate text direction, which was a shortcoming. There were a number of review items and proposals put forth (See [RDF Literals and Base Directions](https://w3c.github.io/rdf-dir-literal/) by Ivan Herman and Pierre-Antoine Champin, for example and further discussion).
```
Comment (TallTed) [inline docs/text-direction.md:7]:
```suggestion
The [_Internationalization Best Practices for Spec Developers_ Note](https://www.w3.org/TR/international-specs/) has a whole section devoted to [text direction](https://www.w3.org/TR/international-specs/#text_direction), and this is a checklist item for the internationalization review all specs need to go through. When this was performed for JSON-LD 1.1, it was noted that the underlying data model (RDF) had no way to indicate text direction, which was a shortcoming. There were a number of review items and proposals put forth (see [_RDF Literals and Base Directions_](https://w3c.github.io/rdf-dir-literal/) by Ivan Herman and Pierre-Antoine Champin, for example and further discussion).
```

File: docs/text-direction.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+
+JSON-LD added support by introducing the [`@direction` keyword](https://www.w3.org/TR/json-ld11/#dfn-base-direction) added to a Literal Object. When serializing to and from an RDF Dataset, non-normative solutions were proposed using [The `i18n` Namespace](https://www.w3.org/TR/json-ld11/#the-i18n-namespace) and [The `rdf:CompoundLiteral`](https://www.w3.org/TR/json-ld11/#the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties) along with associated properties. The idea being that these would be interim solutions available before normative behavior could be described in a future version of RDF.
```
Comment (TallTed) [inline docs/text-direction.md:9]:
```suggestion
JSON-LD added support by introducing the [`@direction` keyword](https://www.w3.org/TR/json-ld11/#dfn-base-direction) added to a Literal Object. When serializing to and from an RDF Dataset, non-normative solutions were proposed using [the `i18n` Namespace](https://www.w3.org/TR/json-ld11/#the-i18n-namespace) and [the `rdf:CompoundLiteral`](https://www.w3.org/TR/json-ld11/#the-rdf-compoundliteral-class-and-the-rdf-language-and-rdf-direction-properties) along with associated properties. The idea was that these interim solutions would be available before normative behavior could be described in a future version of RDF.
```

File: docs/text-direction.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+
+Disadvantages:
+* Introduces a new blank node.
+* Also same dis-advantages as the `i18n` datatype solution.
+
+Also see the [Pros and Cons](https://w3c.github.io/rdf-dir-literal/#pros-and-cons-2) discussion in [RDF Literals and Base Directions](https://w3c.github.io/rdf-dir-literal/),
+
+## Extend `langString`
+
+As expressed in [RDF Literals and Base Directions](https://w3c.github.io/rdf-dir-literal/) [Extend `langString`](https://w3c.github.io/rdf-dir-literal/#extending-lang-string), the definition for `rdf:langString` could be extended to include _both_ the language tag and the text direction by introducing a separator such as a carat (`^`).
```
Comment (TallTed) [inline docs/text-direction.md:71]:
```suggestion
As expressed in [RDF Literals and Base Directions](https://w3c.github.io/rdf-dir-literal/), the [definition for `rdf:langString` could be extended](https://w3c.github.io/rdf-dir-literal/#extending-lang-string) to include _both_ the language tag and the text direction by introducing a separator such as a carat (`^`).
```

**Review by pchampin — APPROVED** [review]

**Pull request merged**